tests/integration/modifiers/key-down-test.js

```diff
@@ -0,0 +1,15 @@
+import { module, test } from 'qunit';
```

Comment:
Can you add a test for the `key-down` modifier and `key-up` modifier? It could be as simple as asserting that they get registered on the `key-manager` service after render.

Reply:
Done. And upgraded testing to typescript